templates/blank-typescript/index.html

```diff
@@ -1,26 +1,18 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Did something change with the way %PUBLIC_URL% works in vite?

Reply:
https://vitejs.dev/guide/#index-html-and-project-root
- URLs inside index.html are automatically rebased so there's no need for special %PUBLIC_URL% placeholders and index.html moved to root